api.planx.uk/helpers.ts

```diff
@@ -1,5 +1,5 @@
 import { gql } from "graphql-request";
-import { capitalize } from "lodash";
+import lodash from "lodash";
```

Comment:
There's a few options here to avoid us having to import all of lodash - 

- Cherry pick import via named path, i.e. `import capitalize from "lodash/capitalize";` ([docs](https://lodash.com/))
- Switch to [lodash-es](https://www.npmjs.com/package/lodash-es)
- Install named functions as packages, i.e. add `lodash/capitalize` to `package.json`

I think `lodash-es` might be the best/simplest option? Happy for this to be a follow up PR or to be picked up here - whatever works best for you.

Reply:
Tried out `lodash-es` but I'm sure it was implemented very well, and it turned out to be easier to go with your first suggestion!